docs/how-to-guides/manage-label-based-access-control-permissions.md

```diff
@@ -0,0 +1,30 @@
+---
+id: manage-label-based-access-control-permissions
+title: How to manage label-based access control permissions (Enterprise)
+sidebar_label: Manage label-based access control permissions
+---
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/security.md)
+
+:::warning
+This is an Enterprise feature.
+If the Memgraph Enterprise is not activated, all the created users will be granted
+all of the privileges.
+Regardless, all of the previously defined privileges will still be applied.
```

Comment:
What do you mean by this last sentence?

Reply:
I actully left this paragraph since it was in the previous how to guide of authorization.
http://localhost:3000/docs/memgraph/next/how-to-guides/manage-user-privileges